modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/network/messages/ClusterStateReceivedMessage.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.cluster.management.network.messages;
+
+import org.apache.ignite.network.NetworkMessage;
+import org.apache.ignite.network.annotations.Transferable;
+
+/**
+ * Successful response for receiving a {@link ClusterStateMessage}.
+ */
+@Transferable(CmgMessageGroup.CLUSTER_STATE_RECEIVED)
+public interface ClusterStateReceivedMessage extends NetworkMessage {
```

Comment:
There's no general OK response yet, since this there are only two OK responses overall and one of them is not redundant. I will remove this class in the future if a general OK response will appear.

Reply:
I'd rather add general OK response right now, and use it here, because it might be not you but someone else who will add similar pseudo ok response one more time. Up to you though.

Reply:
done